match2/commons/match_summary_base.lua

```diff
@@ -13,17 +13,17 @@ end
 local Header = Class.new(
 	function(self)
 		self.root = mw.html.create('div')
-		self.root:addClass('brkts-popup-header')
+		self.root:addClass('brkts-popup-header-dev')
```

Comment:
Why's this?

Reply:
There isn't a way to override this selector:

```
.brkts-popup-header>* {
    display: flex;
    width: 45%
}
```

The workaround is to use a different css class alltogether.

Reply:
Hmm, pity.

Reply:
once we have everything somewhat ready (/but before we push it to the wikis for real) we need to consolidate the css stuff anyways and when doing so we also have to go through all the modules and adjust them accordingly